### PR TITLE
[13.0][MIG] website_sale_product_attribute_filter_visibility: Migration to 13.0

### PR DESCRIPTION
**Pull request by Tardo**
cc @tecnativa TT21703

### CODE REVIEW

File: website_sale_product_attribute_filter_visibility/views/product_attribute_views.xml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        </field>
+    </record>
+
+    <record id="attribute_view_form" model="ir.ui.view">
+        <field name="model">product.attribute</field>
+        <field name="inherit_id" ref="product.product_attribute_view_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//form/sheet/group" position="before">
+                <div class="oe_button_box" name="button_box">
+                    <button name="website_publish_button"
```
Comment (sergio-teruel) [inline website_sale_product_attribute_filter_visibility/views/product_attribute_views.xml]: See now how Odoo adds this smart button here.. 
https://github.com/odoo/odoo/blob/800abfc7baa8461f6b7b5ece95fa7700d3cd872a/addons/website_sale/views/product_views.xml#L91

**Review by sergio-teruel — CHANGES_REQUESTED** [review]

**Comment by sergio-teruel** [comment]
See the website_published button
Here the product's form
Selección_563
And here the attribute's form
Selección_564

**Comment by Tardo** [comment]
Change done! Now uses 'website_publish_button' widget.

**Comment by sergio-teruel** [comment]
@Tardo fix travis.... pre-commit... pre-commit!!! :smile:

**Comment by Tardo** [comment]
Done!

**Review by sergio-teruel — APPROVED** [review]

**Comment by Tardo** [comment]
ping @chienandalu

**Comment by Tardo** [comment]
ping @chienandalu

File: website_sale_product_attribute_filter_visibility/README.rst [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+To use this module you need to:
+
+* Go to 'Sales > Settings > Product Attributes'.
```
Comment (chienandalu) [inline website_sale_product_attribute_filter_visibility/README.rst:41]:
```suggestion
* Go to *Website > Settings > Attributes*.
```

File: website_sale_product_attribute_filter_visibility/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# Copyright 2018 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Website Sale Product Attribute Filter Visibility",
+    "summary": "Allow hide any attributes in shop attributes filter",
+    "version": "13.0.1.0.0",
+    "development_status": "Beta",
```
Comment (chienandalu) [inline website_sale_product_attribute_filter_visibility/__manifest__.py]: Isn't it stable already?

**Review by chienandalu — APPROVED** [review]
Functional tests :+1: 

Some minor and non blocking remarks

**Comment by Tardo** [comment]
Changes done

**Review by chienandalu — APPROVED** [review]

**Comment by chienandalu** [comment]
Thanks :)

**Pull request merged**